### PR TITLE
Added the HG-3 engine (uses the SSTU J-2 mesh, that's been altered by me to look like the HG-3 outline in astronautix.com)

### PR DESCRIPTION
**Pull request by Katniss218**
Some screenshots in-game:
[Imgur](https://i.imgur.com/oxSI7Zn.png)
[Imgur](https://i.imgur.com/HsAU7lz.png)

Please double-check if the engine works (it should, but you never know, if not tested using the repo - my install relies on a different name for this engine (and on SSTU))

### CODE REVIEW

File: GameData/ROEngines/RealPlume/HG3_SSTUKatniss_RealPlume.cfg [hunk 1]
```diff
@@ -0,0 +1,99 @@
+@PART[ROE-HG3]:NEEDS[RealPlume]:BEFORE[RealPlume]
+{
+	@MODULE[ModuleEngineConfigs]
+	{
+		%type = ModuleEnginesRF
```
Comment (DRVeyl) [inline GameData/ROEngines/RealPlume/HG3_SSTUKatniss_RealPlume.cfg:5]: This seems a strange thing to do in a plume patch?

File: GameData/ROEngines/RealPlume/HG3_SSTUKatniss_RealPlume.cfg [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+		@Hydrolox_UpperBlue
+		{
+			|_ = CombinedPlume
+		}
+	}
+	@MODULE[ModuleEngines*]
+	{
+		@name = ModuleEnginesRF
+		%powerEffectName = CombinedPlume
+	}
```
Comment (DRVeyl) [inline GameData/ROEngines/RealPlume/HG3_SSTUKatniss_RealPlume.cfg:98]: Changing the ModuleEngines module type in a plume patch still seems strange.  Setting the powerEffectName is fine.

Add closing linebreak to last line

File: GameData/ROEngines/RealPlume/HG3_SSTUKatniss_RealPlume.cfg (hunk 1 above)
Comment (Katniss218) [reply GameData/ROEngines/RealPlume/HG3_SSTUKatniss_RealPlume.cfg:5]: What seems strange is that I just copied the plume from ROE J-2, so it's there too

File: GameData/ROEngines/RealPlume/HG3_SSTUKatniss_RealPlume.cfg (hunk 2 above)
Comment (Katniss218) [reply GameData/ROEngines/RealPlume/HG3_SSTUKatniss_RealPlume.cfg:98]: Yeah, agree, but the same thing is going on in the J-2s plume, which I obviously haven't touched.

**Pull request merged**